src/app/search/loadouts/search-filters/known-values.ts

```diff
@@ -0,0 +1,29 @@
+import { DestinyClass } from 'bungie-api-ts/destiny2';
+
+export const subclassHashToElementNames: {
+  [key in DestinyClass]: { [hash: number]: string } | undefined;
+} = {
+  [DestinyClass.Warlock]: {
+    2849050827: 'void',
+    3941205951: 'solar',
+    3168997075: 'arc',
+    3291545503: 'stasis',
+    4204413574: 'strand',
+  },
```

Comment:
Do we really not have a way to map subclasses to elements? Even in `subclass.ts`?

Reply:
I didn't find it when I initially looked but I will look again.

Reply:
We do, I wasn't aware of that file so I will move over to using it.

Reply:
Actually, it requires a bit of work since it all expects having a DIMItem available and we are working with `DestinyInventoryItemDefinition`. It isn't too bad though so I will move to this eventually.